### ISSUE TITLE
Use of CString

### ISSUE THREAD

**Issue by virtualritz** [Twinklebear/oidn-rs#9]
I was just wondering. Is there any reason to prefer:
```rust
                let buf_name = CString::new("normal").unwrap();
                unsafe {
                    oidnSetSharedFilterImage(
                        self.handle,
                        buf_name.as_ptr(),
                        ...
```
Over
```rust
                unsafe {
                    oidnSetSharedFilterImage(
                        self.handle,
                        &b"normal\0" as *const _ as _,
                        ...
```

I'm still a Rust n00b but my gut tells me the latter will produce less code/have no runtime overhead.

**Comment by Twinklebear** [comment]
I didn't know about the byte string stuff actually, this might be nicer to migrate too

**Comment by virtualritz** [comment]
I have a PR for you then later today. I also added `set_hdr_scale()` since I needed that.

**Comment by Twinklebear** [comment]
Resolved in #10
